workflow/run_analysis.rb

```diff
@@ -388,6 +392,11 @@ def make_apply_logic_arg(logic)
     options[:debug] = true
   end
 
+  options[:building_ids] = []
+  opts.on('-i', '--building_id ID', Integer, 'Only run this building ID; can be called multiple times') do |t|
```

Comment:
I like this - is there a way to figure out what building id failed in GitHub Actions to more quickly reproduce locally?

Reply:
Good idea. I will update `run_analysis.rb` to report any failed IDs to the screen.